.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -20,10 +20,10 @@ assignees: ''
 <!-- If applicable, add screenshots to help explain your problem. -->
 
 **Environment Info**
-- OS: <!-- [e.g. Linux, Windows 11] -->
-- Trippy version: <!-- [e.g. the output of `trip -V`] -->
-- Installation method: <!-- [e.g. `brew`, `winget`, `cargo`] -->
-- Terminal / Console: <!-- e.g. `iterm2`, `cmd.exe`, `PowerShell` -->
+- OS: <!-- [e.g. Linux, Windows 11, MacOS] -->
```

Comment:
nit: shall we drop the `[` and `]` for consistency?

Reply:
Missed that one thanks, will do